docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
++-------------+--------------------------+-----------------------+----------+
+
+* ``dev`` is for development workflows and uses the ``orgs/dev.json`` configuration file. It has a seven-day lifespan.
+* ``qa`` is for testing workflows. ``qa`` and ``dev`` are the same out of the box, but can be customized to suit the needs of the project. It has a seven-day lifespan.
+* ``feature`` is for continuous integration and uses the ``orgs/dev.json`` configuration file. It has a one-day lifespan.
+* ``beta`` is for continuous integration or hands-on testing. It uses the ``orgs/beta.json`` configuration. It has a one-day lifespan.
+* ``release`` is for continuous integration, hands-on testing, or demo workflow. It uses the ``orgs/release.json`` configuration. It has a one-day lifespan.
+
+If your project has customized the available orgs, you'll see more orgs, or differences in configuration, compared to a new project.
+
+
+
+Create a Scratch Org
 --------------------
 
-Implicit creation of scratch orgs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To create a scratch org from a configuration, use it as the target of a command, task, or flow. CumulusCI automatically initializes orgs when they're first used.
+
+    Example: Create a scratch org from the ``dev`` configuration and review information about the created org.
+
+.. code-block:: console
+
+    $ cci org info dev
+
+..
+
+    When the org is created, it's associated with the name ``dev`` in the CumulusCI keychain and can be used with other commands until it expires. When an org expires or is removed, its associated configuration is left in place, and can be recreated whenever needed.
+
+It's possible to create new orgs in the keychain that inherit their configuration from a built-in org.
+
+    Example: Create a new org that uses the same configuration as type ``dev``, but is still independent of ``dev`` org.  
+    
+.. code-block ::
+
+    $ cci org scratch dev <org>
+    $ cci org info <org>
+    
+You can have as many named orgs as you wish, or none at all. Many CumulusCI users work only with built-in orgs.
+
+
+Scratch Org Limits
+^^^^^^^^^^^^^^^^^^
 
-List orgs
+Each scratch org you create is counted against limits in your DevHub. Scratch orgs count against an *active* scratch org limit, which controls how many orgs can exist at the same time, and a *daily* scratch org limit, which controls how many total orgs can be created per day.
+
+Scratch org limits are based on your DevHub's edition and your Salesforce contract. To review limits and consumption:
+
+.. code-block:: console
+
+    $ sfdx force:limits:api:display -u <username>
+
+``<username>`` is your DevHub username. The limit names are ``ActiveScratchOrgs`` and ``DailyScratchOrgs``.
+
+
+
+
+List Orgs
 ---------
 
-Open an org in the browser
---------------------------
+When inside a project repository, run ``cci org list`` to see all the orgs you have configured or connected.
```

Comment:
This subsection is a bit thin. Doesn't necessarily have to be changed, but just wanted eyes on it for confirmation.